resources/lang/pl.yml

```diff
@@ -343,4 +343,4 @@ LanguageName: "Polish"
 Author: "Nacioszeczek"
 
 # Language file version. Don't touch it!
-Version: 93
\ No newline at end of file
+Version: 4
```

Comment:
Please do not change the version.
The version is shared between all language files and is used by the plugin to determine weather a language file contains all the keys or not (and add the new keys).
Also values bellow 90 will make the plugin interpret the language file as a Marriage Master v1 language file which will result in a messed up language file after loading it.

Reply:
Ok, got it.